pkg/policy.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024 The Witness Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/in-toto/go-witness/dsse"
+	"github.com/in-toto/go-witness/log"
+)
+
+// Load policy from a file or Archivista
+//
+// It prefers to load from a file, if it fails, it tries to load from Archivista
+func LoadPolicy(ctx context.Context, policy string, ac ArchivistaClienter) (dsse.Envelope, error) {
```

Comment:
I am not totally 100% sure on this, but again this _could_ go in `go-witness` rather than in here?

I suppose we don't want people to depend on this the Witness CLI as a library. In which case, we probably would be better off changing `pkg/` to `internal/`. That way, the packages defined inside `internal/` can only be used internally within the witness repo and other people can't depend on the logic 😄.